### PR TITLE
15892: Save feature after geometry edit

### PR DESCRIPTION
**Pull request by geertplaisier**

### CODE REVIEW

File: tailormap-components/projects/core/src/lib/workflow/workflows/EditgeometryWorkflow.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ export class EditgeometryWorkflow extends Workflow {
       this.vectorLayer.readGeoJSON(geom);
```
Comment (mtoonen) [inline tailormap-components/projects/core/src/lib/workflow/workflows/EditgeometryWorkflow.ts]: Isn't the editgeometryworkflow obsolete now that the edit-feature-geometry.service.ts exists?

File: viewer/src/main/webapp/viewer-html/common/viewercontroller/ViewerController.js [hunk 2]
```diff
@@ -469,6 +469,25 @@ Ext.define("viewer.viewercontroller.ViewerController", {
...
+                fillcolor: "FF0000",
+                fillopacity: 50,
+                strokecolor: "FF0000",
+                strokeopacity: 50
+            }
+        });
+        this.mapComponent.getMap().addLayer(layer);
+        return layer;
+    },
+
```
Comment (mtoonen) [inline viewer/src/main/webapp/viewer-html/common/viewercontroller/ViewerController.js:490]: Hmm, not entirely happy with this, but as the ext side will be removed, lets not split hair ;-)
Probably did this to create better readable code in angular, so I think it's acceptable

**Review by mtoonen — APPROVED** [review]
Looks good, but I think editgeometryworkflows.ts can be removed, no?

File: viewer/src/main/webapp/viewer-html/common/viewercontroller/ViewerController.js (hunk 2 above)
Comment (geertplaisier) [reply viewer/src/main/webapp/viewer-html/common/viewercontroller/ViewerController.js:490]: Yep, I wanted to keep the specifics of how to create a layer in the 'old' code so edit-geometry-services does not need to know how to create one.

File: tailormap-components/projects/core/src/lib/workflow/workflows/EditgeometryWorkflow.ts (hunk 1 above)
Comment (geertplaisier) [reply tailormap-components/projects/core/src/lib/workflow/workflows/EditgeometryWorkflow.ts]: Removed

**Pull request merged**